android/app/src/main/java/io/imagekit/imagekitdemo/MainActivity.kt

```diff
@@ -0,0 +1,48 @@
+package io.imagekit.imagekitdemo
+
+import android.content.Intent
+import android.os.Bundle
+import androidx.appcompat.app.AppCompatActivity
+import com.facebook.drawee.backends.pipeline.Fresco
+import com.imagekit.android.ImageKit
+import com.imagekit.android.entity.TransformationPosition
+import io.imagekit.imagekitdemo.databinding.ActivityMainBinding
+
+class MainActivity : AppCompatActivity() {
+    private var binding: ActivityMainBinding? = null
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        binding = ActivityMainBinding.inflate(layoutInflater)
+        setContentView(binding?.root)
+
+        ImageKit.init(
+            context = applicationContext,
+            publicKey = "IK_PUBLIC_KEY",
+            urlEndpoint = "https://ik.imagekit.io/IMAGEKIT_ID",
+            transformationPosition = TransformationPosition.PATH,
+        )
```

Comment:
Lets add a comment before this `Initialize the ImageKit SDK with your credentials and configurations.`

Reply:
Done. Please check